BoxOffice/DTO/BoxOfficeDTO.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  BoxOfficeDTO.swift
+//  BoxOffice
+//
+//  Created by Sunny on 2023/04/25.
+//
+
+import Foundation
+
+struct BoxOfficeDTO: Decodable {
+    
+    let boxOfficeResult: BoxOfficeResultDTO
```

Comment:
p4;

```suggestion
    let result: BoxOfficeResultDTO
```

BoxOfficeDTO 타입 내부의 프로퍼티이기 때문에 `result` 로만 이름을 지어도 의미를 전달하는 데 무리가 없을 거 같은데 어떻게 생각하실까요? 👀

Reply:
DTO 타입으로 데이터를 사용할 때 어떤 맥락으로 표현되는지를 고민 후 `result`로만 지어도 명확하다고 생각하였습니다!
해당 부분을 수정하며 `BoxOfficeResultDTO` 타입의 프로퍼티 중 박스오피스 타입, 박스오피스 조회 기간에 해당하는 부분도 `type`, `inqueryRange`로 변경하였고, 일간 박스오피스를 나타내는 `dailyBoxOffices`에는 키워드가 필요하다고 판단하여
아래 커밋에 반영하였습니다!

> 해당 커밋: 245832d